### ISSUE TITLE
Extend the includes transform to support filter.length

### ISSUE THREAD

**Issue by aboyton** [lebab/lebab#218]
The `includes` transformation converts `array.indexOf` to `includes`.

Another style I've seen is

```js
if (array.filter(foo => foo === bar).length) { /* */ }
```

which can be converted to

```js
if (array.includes(bar)) { /* */ }
```

I'd propose that we extend the `includes` transformation to include this case.

**Comment by nene** [comment]
Hi, @aboyton. Can you provide some references that would show how widespread this style is. Like links to more than one Github repo that use this style. I've personally never seen such style used.

**Comment by nene** [comment]
Ah, I see you already created a pull request as well.

However I'm still interested in how widespread this pattern is. I'm not too keen on accepting a fairly large chunk of code that I'm not sure solves a common-enough problem.

I should also note that the implementation only covers the case of it being used with arrow functions, not with normal functions.

I do appreciate the effort you've put into this.

**Comment by nene** [comment]
Given no further input, I'm closing this.

**Comment by aboyton** [comment]
Sorry about not getting back to you earlier. I've had less free time to play with this since my second kid was born.

Very happy for this not to land. I had a colleague who was writing in that style and so I figured I'd see how hard it was to get lebab to support it. Since I'd gone to the effort I figured I may as well submit a pull request in case you were interested, but it definitely complicated the code more than I was expecting for an uncommon coding style, so I'm really not fussed if this isn't merged.
